packages/block-library/src/post-template/index.php

```diff
@@ -45,9 +45,13 @@ function block_core_post_template_uses_featured_image( $inner_blocks ) {
  * @return string Returns the output of the query, structured using the layout defined by the block's inner blocks.
  */
 function render_block_core_post_template( $attributes, $content, $block ) {
-	$page_key            = isset( $block->context['queryId'] ) ? 'query-' . $block->context['queryId'] . '-page' : 'query-page';
-	$enhanced_pagination = isset( $block->context['enhancedPagination'] ) && $block->context['enhancedPagination'];
-	$page                = empty( $_GET[ $page_key ] ) ? 1 : (int) $_GET[ $page_key ];
+	$page_key                        = isset( $block->context['queryId'] ) ? 'query-' . $block->context['queryId'] . '-page' : 'query-page';
+	$enhanced_pagination             = isset( $block->context['enhancedPagination'] ) && $block->context['enhancedPagination'];
+	$interactivity_api_router_config = wp_interactivity_config( 'core/router' );
+	if ( ! empty( $interactivity_api_router_config ) && ! empty( $interactivity_api_router_config['fullClientSideNavigation'] ) ) {
```

Comment:
`empty` is a language construct and it performs internally all necessary checks:
```suggestion
	if ( ! empty( $interactivity_api_router_config['fullClientSideNavigation'] ) ) {
```

The same applies to all other occurrences.

Reply:
Changed in [this commit](https://github.com/WordPress/gutenberg/pull/59707/commits/a26bfa70e582d5b06d186394301a9748db8a90eb).